docs/changelog/1602.md

```diff
@@ -1 +1 @@
-- Add support for waveform iterations for the serial-explicit coupling.
+- Add support for waveform iterations for the serial-explicit coupling. The `second` participant can now use higher order time interpolation.
```

Comment:
```suggestion
- Add support for waveform iterations for the serial-explicit coupling. The `second` participant can now use higher-order time interpolation.
```
@uekerman ? This still confuses me...

Reply:
I would write "higher-order time interpolation" as it is an adjective, but I did not do a deep dive here :grin:

Reply:
I use that.